src/pages/Proposal/ProposalSummary/index.tsx

```diff
@@ -1,15 +1,135 @@
 /** @jsxImportSource @emotion/react */
 import React from 'react';
-import { Paper } from '@mui/material';
+import { Paper, Typography } from '@mui/material';
+import { ActiveChip, BscLink, Chip, Countdown, PrimaryButton, SecondaryButton } from 'components';
+import { IProposal } from 'types';
+import { useTranslation } from 'translation';
+import Stepper from './Stepper';
 import { useStyles } from './styles';
 
-interface IProposalSummary {
+interface IProposalSummaryUiProps {
   className?: string;
+  proposal: IProposal;
 }
 
-export const ProposalSummary: React.FC<IProposalSummary> = ({ className }) => {
+interface IProposalSummaryContainerProps {
+  cancelProposal: () => void;
+  executeProposal: () => void;
+  queueProposal: () => void;
+}
+
+export const ProposalSummaryUi: React.FC<
+  IProposalSummaryUiProps & IProposalSummaryContainerProps
+> = ({ className, proposal, cancelProposal, queueProposal, executeProposal }) => {
   const styles = useStyles();
-  return <Paper css={styles.root} className={className} />;
+  const { t } = useTranslation();
+  const {
+    state,
+    id,
+    description: { title },
+    createdDate,
+    createdTxHash,
+    startDate,
+    startTxHash,
+    cancelDate,
+    cancelTxHash,
+    queuedDate,
+    queuedTxHash,
+    executedDate,
+    executedTxHash,
+    endTxHash,
+    endDate,
+  } = proposal;
+
+  let updateProposalButton = null;
```

Comment:
As a rule of thumb, I try to avoid using `null` as much as possible because of all its weird behaviors (like `typeof null === true` returned `true` 😅).
You can use undefined:
```suggestion
  let updateProposalButton: React.ReactNode | undefined;
```

Reply:
You can render undefined which is why I chose null here